platform/runner-extension/plugin/std/sys/src/main/resources/sys/extension.lua

```diff
@@ -98,14 +98,17 @@ function extension()
         function export.func.list(query)
             query = query or {}
             query.namespace_ids = query.namespace_ids or {}
-
             return internal.func_list(query)
         end
 
         function export.func.invoke(func_id, body)
             return internal.func_invoke(func_id, body)
         end
 
+        function export.func.update(body)
+            return internal.func_update(body)
```

Comment:
this can blow up in your FuncUpdateFunction if body is nil
```suggestion
        function export.func.update(body)
            body = body or {}
            return internal.func_update(body)
```